crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -72,7 +89,7 @@ impl CachedPipelineState {
 }
 
 #[derive(Default)]
-pub struct ShaderData {
+struct ShaderData {
```

Comment:
This change shouldn't be part of a docs PR.

Reply:
Reverted.

Reply:
Actually, opened a separate PR for this #5609